src/ui/Analytics/LiveAnalyticsClient.ts

```diff
@@ -343,7 +343,7 @@ export class LiveAnalyticsClient implements IAnalyticsClient {
     let modifiedMeta: IChangeableAnalyticsMetaObject = _.extend({}, meta);
     modifiedMeta['JSUIVersion'] = version.lib + ';' + version.product;
 
-    if (result) {
+    if (!(modifiedMeta['contentIDKey'] && modifiedMeta['contentIDValue']) && result) {
```

Comment:
I would find it more readable if the condition was extracted into a well-named variable. e.g.

```
const hasContentIDKeyAndValue = modifiedMeta['contentIDKey'] && modifiedMeta['contentIDValue'];

if (result && !hasContentIDKeyAndValue) {
    ....
}
```